inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/activation/activation_kernel_base.h

```diff
@@ -68,5 +68,8 @@ class ActivationKernelBase : public KernelBaseOpenCL {
     virtual JitConstants GetJitConstants(const activation_params& params, DispatchData dispatchData) const;
     virtual DispatchData SetDefault(const activation_params& arg) const;
     KernelsData GetCommonKernelsData(const Params& params, const optional_params& options) const;
+    KernelsPriority GetKernelsPriority(const Params& /*params*/, const optional_params& /*options*/) const {
```

Comment:
Why do you need this method in the `base` classes for operations? It's already defined in the `KernelBase` class:
```
    virtual KernelsPriority GetKernelsPriority(const Params& /*params*/,
                                               const optional_params& /*options*/) const {
        return DONT_USE_IF_HAVE_SOMETHING_ELSE;
    };
```

Reply:
I figured that it would be more clear. Is it better to remove this?

Reply:
Yes, it's completely useless method in the base op classes as you have dedicated implementation in each actual kernel class. And you defined it only for part a part of such classes, so in addition it's inconsistent right now.